modules/src/ics04_channel/error.rs

```diff
@@ -12,6 +13,14 @@ use tendermint_proto::Error as TendermintError;
 define_error! {
     #[derive(Debug, PartialEq, Eq)]
     Error {
+        Ics02Client
+            [ client_error::Error ]
+            | _ | { "ics02 client error" },
+
```

Comment:
```suggestion
```
I would suggest removing this variant from here and reusing `Ics03Connection::Ics02Client`.